libpod/container.go

```diff
@@ -469,7 +472,44 @@ func (c *Container) Unpause() error {
 // Export exports a container's root filesystem as a tar archive
 // The archive will be saved as a file at the given path
 func (c *Container) Export(path string) error {
-	return ErrNotImplemented
+	c.lock.Lock()
+	defer c.lock.Unlock()
+
+	if !c.valid {
+		return errors.Wrapf(ErrCtrRemoved, "container %s is not valid", c.ID())
+	}
+
+	if err := c.runtime.state.UpdateContainer(c); err != nil {
+		return err
+	}
+
+	mountPoint := c.state.Mountpoint
+	if mountPoint == "" {
```

Comment:
Should just test c.state.Mounted instead

Reply:
fixed